src/app/templates/scripts/_server.js

```diff
@@ -67,17 +68,29 @@ const bundlers = {
 };
 
 const bundle = (name) => {
-  return new Promise((resolve, reject) => {
-    bundlers[name]
-      .bundle()
-      .pipe(fs.createWriteStream(dests[name]))
-      .on('finish', resolve)
-      .on('error', reject);
-  });
+  if (name === 'lang') {
+    vjslangs(srces.langs, dests.langs);
+  } else if (name === 'sass') {
+    fs.writeFileSync(
+      dests.css,
+      sass.renderSync({file: srces.css, outputStyle: 'compressed'}).css
+    );
+  } else if (bundlers.hasOwnProperty(name)) {
+    return new Promise((resolve, reject) => {
+      bundlers[name]
+        .bundle()
+        .pipe(fs.createWriteStream(dests[name]))
+        .on('finish', resolve)
+        .on('error', reject);
+    });
+  }
 };
 
 mkdirp.sync('dist');
 
+<% if (lang) { %>bundle('lang');<% } -%>
+<% if (sass) { %>bundle('sass');<% } -%>
```

Comment:
Well, the reason is that they're called in two places and I like the idea of each of them being called through the same function. I have a better abstraction in mind, though.